github/Requester.py

```diff
@@ -614,6 +615,22 @@ def __structuredFromJson(self, data: str) -> Any:
                     raise
                 return {"data": data}
 
+    def requestJsonAndValidateStatus(
+        self,
+        verb: str,
+        url: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, Any]] = None,
+        input: Optional[Any] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+        valid_codes: Optional[Set[int]] = None,
```

Comment:
This should be required parameter I think.

Reply:
Sure, just modified.